maestrowf/conductor.py

```diff
@@ -190,14 +194,14 @@ def main():
 
         cancel_lock_path = os.path.join(args.directory, ".cancel.lock")
         logger.info("Starting to monitor '%s'", dag.name)
-        monitor_study(dag, study_pkl[0], cancel_lock_path, args.sleeptime)
+        study_status = monitor_study(dag, study_pkl[0], cancel_lock_path, args.sleeptime)
```

Comment:
This line fails flake8 checks -- it's longer than 79 characters.

Reply:
I'll take a look